meta-resin-common/recipes-support/resin-conf-reset/resin-conf-reset/resin-conf-reset

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -e
```

Comment:
It would probably be easier, but this way we make sure that we create the correct structure without having to rely on a different component to be synced with our bind mounts (the rest of the read-only mechanism in general)